CRM/Event/Form/SelfSvcUpdate.php

```diff
@@ -108,6 +108,12 @@ class CRM_Event_Form_SelfSvcUpdate extends CRM_Core_Form {
    */
   protected $_details = array();
   /**
+   * Is backoffice form?
+   *
+   * @array bool
+   */
+  protected $_isBackoffice = FALSE;
+  /**
```

Comment:
if there is not another pattern we are trying to match $isBackOffice (caps on Office, no _) is preferable